ground_truth_slam/plugin.cpp

```diff
@@ -36,16 +36,16 @@ class ground_truth_slam : public plugin {
 			return;
 		}
 
-		pose_type* true_pose = new pose_type{_m_sensor_data_it->second};
+		pose_type* true_pose = new (_m_true_pose.allocate()) pose_type{_m_sensor_data_it->second};
```

Comment:
Allocate a `new` `pose_type` using the memory-space from `_m_true_pose.allocate()`. It's called a "placement new", which is like `new` but reuses existing space instead of allocating.